streams/src/test/java/org/apache/kafka/streams/tests/StreamsUpgradeTest.java

```diff
@@ -158,7 +159,8 @@ public ByteBuffer subscriptionUserData(final Set<String> topics) {
                     LATEST_SUPPORTED_VERSION + 1,
                     taskManager.processId(),
                     userEndPoint(),
-                    taskManager.getTaskOffsetSums()
+                    taskManager.getTaskOffsetSums(),
+                    Bytes.EMPTY
```

Comment:
We should do the same thing that we do in StreamsPartitionAssignor here, ie a changing byte. This is supposed to be like the same exact thing but for a "future version" of the partition assignor, to test version probing and general compatibility. 
It's a bit annoying that you have to just copy the same exact stuff but that's just how it is for now